pyproject.toml

```diff
@@ -77,6 +78,8 @@ _format_isort = "isort ."
 _format_black = "black ."
 tests = {sequence = ["tests_unit"], help = "Run all available tests"}
 tests_unit = {cmd = "pytest --cov --cov-report html --cov-report term -sv tests/unit", help = "Run only unit tests"}
+create_assets_goth = "python -m goth create-assets tests/goth_tests/assets"
+tests_goth = "pytest -svx tests/goth_tests --disable-pytest-warnings --config-override docker-compose.build-environment.use-prerelease=true --config-path tests/goth_tests/assets/goth-config-testing.yml --ssh-verify-connection"
```

Comment:
@approxit what convencion will you use for yapapi in https://github.com/golemfactory/yagna-sdk-team/issues/237?

Reply:
I'm using generic names: `tests_integration` and `tests_integration_assets`.